Examples/Fault/VHT_MPS2_Cortex-M7/README.md

```diff
@@ -69,7 +76,7 @@ The fault triggering is done by entering a number via simulator console (see pos
   - 5: trigger the undefined instruction Usage fault
   - 6: trigger the divide by 0 Usage fault
 
-### Running the example in the uVision
+### Running the example in the uVision (using Arm Compiler 6 toolchain)
```

Comment:
I would remove the comment in bracket.
The .cprj can be for Arm Compiler 6 or GCC, depending on the csolution convert command.